### PR TITLE
Update sample to use compiled framework instead of source files

### PR DESCRIPTION
**Pull request by samgreen**

### CODE REVIEW

File: HelloMixpanel/HelloMixpanel/ViewController.m [hunk 1]
```diff
@@ -159,7 +153,7 @@ - (IBAction)changeColor
     FCColorPickerViewController *colorPicker = [[FCColorPickerViewController alloc]
                                                 initWithNibName:@"FCColorPickerViewController"
                                                 bundle:[NSBundle mainBundle]];
-    colorPicker.color = [UIColor mp_applicationPrimaryColor];
+    colorPicker.color = [UIColor redColor];
```
Comment (yarneo) [inline HelloMixpanel/HelloMixpanel/ViewController.m]: is this intentional?

File: HelloMixpanel/HelloMixpanel/ViewController.m [hunk 2]
```diff
@@ -85,7 +79,7 @@ - (IBAction)setPeopleProperties:(id)sender
 
 - (IBAction)setNotificationType:(id)sender
 {
-    NSArray *types = @[MPNotificationTypeTakeover, MPNotificationTypeMini];
+    NSArray *types = @[@"takeover", @"mini"];
```
Comment (yarneo) [inline HelloMixpanel/HelloMixpanel/ViewController.m]: are the aliases redundant?

**Comment by yarneo** [comment]
Other than the comments, if it compiles well then it looks good to me!

File: HelloMixpanel/HelloMixpanel/ViewController.m (hunk 1 above)
Comment (samgreen) [inline HelloMixpanel/HelloMixpanel/ViewController.m]: Yes, this header is longer exposed. This only sets the initial color of the picker.

File: HelloMixpanel/HelloMixpanel/ViewController.m (hunk 2 above)
Comment (samgreen) [inline HelloMixpanel/HelloMixpanel/ViewController.m]: The aliases are not exposed to the users of the framework. These strings came from the `Mixpanel.h` header.

**Pull request merged**